docker.Makefile

```diff
@@ -46,6 +46,10 @@ clean: build_docker_image
 test-unit: build_docker_image
 	docker run --rm $(ENVVARS) $(MOUNTS) $(DEV_DOCKER_IMAGE_NAME) make test-unit
 
+# run go test
+.PHONY: test
+test: test-unit
```

Comment:
This should be an alias for running both test suites I think

```
.PHONY:
test: test-unit test-e2e
```

The comment can be removed, I don't think it adds anything

Reply:
@dnephin Removed

Reply:
Thanks! How about the other part (aliasing both test suite) ?